officeInsightIndex.js

```diff
@@ -113,11 +116,24 @@ function handleRoomIntentDirectionRequest(intent, context) {
 }
 
 function getMeetingRoom(room, context) {
-  if (directions.Fifty_Three.hasOwnProperty(room)) {
-    createResponse(null, room + " is located on the " + directions.Fifty_Three[room] + " of the Fifty Third Floor", context);
-  } else if (directions.Fifty_One.hasOwnProperty(room)) {
-    createResponse(null, room + " is located on the " + directions.Fifty_One[room] + " of the Fifty First floor", context);
-  }
+  var headers = {
+    'id' : room.toLowerCase(),
+    'anchor' : 'east-lobby'
+  };
+  http.request(createGTFORequest('POST', GTFO_PING, headers), function (response) {
+    response.on('data', function () {
+      var response = room + " is located on the ";
+      if (directions.Fifty_Three.hasOwnProperty(room)) {
+        response += directions.Fifty_Three[room] + " of the Fifty Third floor.";
+      } else if (directions.Fifty_One.hasOwnProperty(room)) {
+        response += directions.Fifty_One[room] + " of the Fifty First floor.";
+      }
+      if (true) {
```

Comment:
Should've probably added a comment around this. Basically, this bit should be configurable, as all alexa's won't necessarily be integrated with the visual map/pings